src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2903,6 +2903,10 @@ public boolean terminate(AbstractParticipant participant,
 
             if (octo)
             {
+                if (participant != null)
+                {
+                    participant.setChannelAllocator(null);
```

Comment:
I'm wondering why is it only in the Octo case?

Reply:
We cancel the channel allocator for the non-octo case [here](https://github.com/jitsi/jicofo/blob/e8d68b021ea84ec18ebe09770d9f518b1e1ed4ec/src/main/java/org/jitsi/jicofo/Participant.java#L530).

Reply:
I can't follow all the logic, but I think the leak still exists: if the octo participant is terminated while the channel allocator is running, the channels it eventually allocates will not be released: https://github.com/jitsi/jicofo/blob/master/src/main/java/org/jitsi/jicofo/AbstractChannelAllocator.java#L169

Reply:
is it that the octo channels never expire?

Reply:
what  I mean is that for regular participants they eventually expire and that was not a big deal

Reply:
Yes, currently the octo channels/endpoints on the bridge never expire. We should definitely fix that. 
Do you think if we expire them on the bridge, then we can leave jicofo as it is? If so, do we still need this PR?

Reply:
either that or extra step that would expire channels if the allocator is canceled

Reply:
in general I think it's better to expire them explicitly in jicofo even fi the bridge will be smar  enough  to expire